tests/5.1/env_var/test_omp_places_env_ll_caches.c

```diff
@@ -17,10 +17,10 @@
 int test_places(){
 	int errors = 0;
 	int test = 1;
-	char* ret_val = ""; 
+	char* ret_val = NULL;
 	setenv("OMP_PLACES", "ll_caches", 1 );
```

Comment:
As/if setting the env via the Makefile works, I think the following makes sense:
```suggestion
```